galsim/detectors.py

```diff
@@ -275,6 +275,62 @@ def applyIPC(self, IPC_kernel, edge_treatment='extend', fill_value=None, kernel_
     else:
         self.array[:,:] = out_array
 
+def applyPersistence(self,imgs,coeffs):
+    """
+    Applies the effects of persistence to the Image instance.
+
+    Persistence refers to the retention of a small fraction of the signal after resetting the
+    imager pixel elements. The persistence signal of a previous exposure is left in the pixel even
+    after several detector resets. This effect is most likely due to charge traps in the material.
+    Laboratory tests on the WFIRST CMOS detectors show that if exposures and readouts are taken in
+    a fixed cadence, the persistence signal can be given as a linear combination of prior pixel
+    values that can be added to the current image.
+
+    This routine takes in one or multiple Image instances and adds them to Image weighted by the
+    values passed on to 'coeffs'. This routine does NOT keep track of realistic dither patterns.
+    During the image simulation process, the user has to queue a list of previous Image
+    instances (img_list) outside the routine by inserting the latest image in the beginning of the
+    list and deleting the oldest image. The values in 'coeffs' tell how much of each Image is to
+    be added. This usually remains constant in the image generation process.
+
+    Calling
+    -------
+
+        >>> img.applyPersistence(imgs=img_list, coeffs=coeffs_list)
+
+    @ param imgs:       an Image instance or a list of previous Image instances that still persist
```

Comment:
Should change format of how parameters are listed to match the signature elsewhere in GalSim.